site_root/input/resources/examples/Condition-cd-001.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
Can you rename this file and the `id` to incorporate no-phi (e.g. cd-001-no-phi)? Can you also make another Condition example called `cd-001` which includes some example PHI?